variables.tf

```diff
@@ -597,3 +597,23 @@ variable "kdump_crash_kernel_memory" {
   description = "The crashkernel memory reservation for kdump occurs during the system boot"
   default     = "2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G"
 }
+
+################################################################
+# Secure NFS variables
```

Comment:
Nit:
```suggestion
# Secure service variables
```

Reply:
Hey @yussufsh,
I have made the changes. Please review at your convenience.
Thank-you.